### PR TITLE
Verrazzano patch release 1.3.1

### PR DESCRIPTION
**Pull request by shashankkumarIITB**
- Updated the docs for Verrazzano patch release 1.3.1

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ linkTitle: Release Notes
 weight: 13
 draft: false
 ---
+### v1.3.1
+Fixes:
+
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:9]: Please put a period at the end of each sentence.

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -4,6 +4,17 @@ linkTitle: Release Notes
...
+- Resolved an issue where Verrazzano uninstall deleted additional namespaces when deleting Rancher components
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:10]: where Verrazzano uninstall -> where the Verrazzano uninstall

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -4,6 +4,17 @@ linkTitle: Release Notes
...
+- Added liveness probe for the AuthProxy NGINX server
+- Added support for using Services as Workloads
+- Added support for arbitrary Helm values overrides
+- Added support for post-install Helm upgrades to ConfigMaps, Secrets, and values in the Verrazzano CR
+- Added support for JWT authentication and authorization policy specification for an applicaiton
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:15]: applicaiton (typo) -> application

Comment (shashankkumarIITB) [reply content/en/docs/releasenotes/_index.md]: Done.

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -4,6 +4,17 @@ linkTitle: Release Notes
 weight: 13
 draft: false
 ---
+### v1.3.1
+Fixes:
+
+- Resolved an issue where Verrazzano uninstall deleted additional namespaces when deleting Rancher components
+- Added liveness probe for the AuthProxy NGINX server
+- Added support for using Services as Workloads
```
[the remark this answers is not in the document]
Comment (shashankkumarIITB) [reply content/en/docs/releasenotes/_index.md]: Done.

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (shashankkumarIITB) [reply content/en/docs/releasenotes/_index.md:9]: Done.

File: content/en/docs/releasenotes/_index.md [hunk 5]
```diff
@@ -7,13 +7,13 @@ draft: false
...
-- Added liveness probe for the AuthProxy NGINX server
-- Added support for using Services as Workloads
-- Added support for arbitrary Helm values overrides
-- Added support for post-install Helm upgrades to ConfigMaps, Secrets, and values in the Verrazzano CR
-- Added support for JWT authentication and authorization policy specification for an applicaiton
-- Added support for Prometheus ServiceMonitor and PodMonitor CRs using Prometheus Operator
+- Resolved an issue where the Verrazzano uninstall deleted additional namespaces when deleting Rancher components.
+- Added liveness probe for the AuthProxy NGINX server.
+- Added support for using Services as workloads.
+- Added support for post install updates to arbitrary helm value overrides sources such as ConfigMaps, Secrets and Values referenced in the Verrazzano CR.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:13]: post install updates -> post-installation updates
helm value -> Helm value
sources -> sources, (comma)
Secrets -> Secrets, (comma)

**Review by rosemarymarano — COMMENTED** [review]
Few more to go.

**Review by rosemarymarano — APPROVED** [review]
I'll approve but if you have further additions, please let me review them. Thank you.

**Review by rosemarymarano — APPROVED** [review]

**Review by rosemarymarano — COMMENTED** [review]
one edit

**Pull request merged**